content/en/docs/Core concepts/concurrency.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Tuning concurrency"
+date: 2020-11-02
+weight: 3
+---
+
+Baker filters and ouput can be configured with a high concurrency profile (input doesn't support
+parallel processing).
+
+This means that those components will process Records in parallel during the pipeline,
+improving performances but adding a small cons: Records order isn't guaranteed anymore
+(due to possible different speed processing between concurrent components).
+
+{{% alert title="Default behaviour" color="primary" %}}
+The default behaviour of Baker is to use concurrency for both filters and output.
+{{% /alert %}}
+
+### Concurrent filtering
```

Comment:
```suggestion
### Filter chain concurrency
```

Emphasize on the fact that a filter chain is a **synchronous** list of filters. But we can add more filter chain and so the filter chains as a whole become concurrent.